src/phoenix/server/app.py

```diff
@@ -182,8 +182,11 @@ def _lifespan(
 ) -> StatefulLifespan[Starlette]:
     @contextlib.asynccontextmanager
     async def lifespan(_: Starlette) -> AsyncIterator[Dict[str, Any]]:
-        async with BulkInserter(db, initial_batch_of_spans) as queue_span:
-            yield {"queue_span_for_bulk_insert": queue_span}
+        async with BulkInserter(db, initial_batch_of_spans) as (queue_span, queue_evaluation):
```

Comment:
can we also pipe in the initial evals from the fixture?
```suggestion
        async with BulkInserter(db, initial_batch_of_spans, inital_batch_of_evaluations) as (queue_span, queue_evaluation):
```

Reply:
I need to first add support for other kinds of evaluations (e.g., document and trace evaluations). Filing this as a ticket.